frontend/ui/elements/screen_fullscreen_menu_table.lua

```diff
@@ -0,0 +1,36 @@
+local _ = require("gettext")
```

Comment:
The coding style says _ goes under those.

Reply:
I've glanced at https://github.com/koreader/koreader-base/wiki/Coding-style 
but still don't understand what you mean.

Reply:
```lua
local Class = require("class")  -- Uppercase class
local Device = require("device")  -- Uppercase class
local func = require("func")  -- Lowercase function
local glob = require("glob")  -- Lowercase function
local _ = require("gettext")  -- Symbol
local T = require("ffi/util").template  -- One of the functions in a file
local V = require("utils2").version  -- The other function, ordered by ASCII.
```

Reply:
Order changed as you wanted.